### PR TITLE
PMs unable to see posts on projects - 3922696672

### PR DESCRIPTION
**Pull request by brentkulwicki**
┆Issue is synchronized with this [Monday item](https://seed-company-squad.monday.com/boards/3451697530/pulses/4836714586) by [Unito](https://www.unito.io)

### CODE REVIEW

File: src/components/post/post.service.ts [hunk 1]
```diff
@@ -67,34 +76,53 @@ export class PostService {
...
+      parent.__typename as 'Postable',
+    );
+    return this.privileges.for(session, parentType, parent).forEdge('posts');
   }
 
   async update(input: UpdatePost, session: Session): Promise<Post> {
     const object = await this.readOne(input.id, session);
 
     const changes = this.repo.getActualChanges(object, input);
+    this.privileges.for(session, Post, object).verifyChanges(changes);
```
Comment (brentkulwicki) [inline src/components/post/post.service.ts:117]: @CarsonF - Mind feedback on this specifically? App seems to be working as expected, but I am like 75% confident in this line and then where I also call the privileges in `delete` (line 125)

Comment (brentkulwicki) [reply src/components/post/post.service.ts:117]: Actually... should this be replaced by the `getPermissionsFromResource`?

Comment (CarsonF) [reply src/components/post/post.service.ts:117]: > Actually... should this be replaced by the `getPermissionsFromResource`?

Yes

File: src/components/post/dto/postable.dto.ts [hunk 2]
```diff
@@ -10,19 +10,22 @@ import { Post } from './post.dto';
   description: stripIndent`
     An object that can be used to enable Post discussions on a Node.
   `,
+  implements: Resource,
 })
-export abstract class Postable {
+export abstract class Postable extends Resource {
```
Comment (CarsonF) [inline src/components/post/dto/postable.dto.ts:15]: I'd like to avoid this if possible. Why? Idk just seems like an unrelated change. Do you have a reason or were you just matching comments?

File: src/components/post/post.service.ts (hunk 1 above)
Comment (brentkulwicki) [reply src/components/post/post.service.ts:117]: Just a heads up. Fixed delete. Update need a little more love tonight or tomorrow.

**Comment by brentkulwicki** [comment]
https://www.loom.com/share/b2b51a832bb8463e83c0a6bd74bbff16?sid=7b760293-4ca5-4dc4-b1de-8e512c9d3d45

**Comment by CarsonF** [comment]
Superceded by #2867
